Gemfile

```diff
@@ -64,6 +64,7 @@ gem 'wkhtmltopdf-binary'
 gem 'foreigner'
 gem 'immigrant'
 gem 'roo', '~> 2.7.0'
+gem 'roo-xls'
```

Comment:
We need to ensure that at least a bundle update does not introduce a version with backward incompatible changes.